Help/About: Initial pass of 6.2 About page

ryelle opened the pull request:
This is a first pass at getting the About strings in, updating the header assets, and updating the style and colors. It sounds like the content is ready-enough to commit with translation wrappers, so I've done that, but some things will likely change in the time before RC1.

I'd appreciate a review from @jpantani or @thetinyl to make sure I've copied over the content correctly — you can review the screenshot below if you don't have a local environment set up.

**Screenshots:**

About page (full page)

![about-full](https://user-images.githubusercontent.com/541093/221761019-357bbb3d-1715-4507-b647-39d065300e55.png)

Credits (only header changed)

![credits](https://user-images.githubusercontent.com/541093/221761020-a1ab161e-6a84-43b9-abb9-14bd3f64cb10.png)

Freedoms (only header changed)

![freedoms](https://user-images.githubusercontent.com/541093/221761021-4a12c4ab-c683-4c5c-ac85-6c7bd8302bd5.png)

Privacy (only header changed)

![privacy](https://user-images.githubusercontent.com/541093/221761024-1f638a4d-617d-4962-8e33-3d9cd27676a0.png)

Trac ticket: https://core.trac.wordpress.org/ticket/57477

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

ryelle committed:
Help/About: Initial pass of 6.2 About page

thetinyl commented:
@ryelle I think it looks good. I'm noticing two updates needed:

- Remove the headline above the Style Book, Copy and paste, Custom CSS section (this change is now reflected in the copy document, too)
- The "Learn more about 6.2" bottom section needs a little bit of tweaking—see screenshot below from Figma design:

<img width="596" alt="image" src="https://user-images.githubusercontent.com/100625704/222024188-e87d2d85-4f4d-4987-88ed-0332325100c9.png">

ryelle commented:
@thetinyl Is the difference in the "Learn more" section the vertical alignment? I did leave all the strings here alone, since this way the current translations can be reused (they've been consistent for the past few releases, I think), but if the copy change is also important I can do that too.

Edit: Ah, I missed the icons, too. Still have the question about the copy.

thetinyl commented:
> I did leave all the strings here alone, since this way the current translations can be reused (they've been consistent for the past few releases, I think), but if the copy change is also important I can do that too.

The copy is actually very similar to what it's been, just condensed a little. It was adjusted to make the resource links stand out a little better. Though from a translation perspective it's probably different enough that it would probably need another look.

ryelle committed:
Updates based on feedback

ryelle commented:
I've removed the heading, fixed the alignment, and added in the icons. The bottom section now looks like this:

![Screen Shot 2023-02-28 at 23 33 58](https://user-images.githubusercontent.com/541093/222045543-2c527590-417a-4ca4-b7c7-74faaa288584.png)

ryelle committed:
Update version on background SVGs

ryelle committed:
Switch from SVG background to real text for translations

audrasjb commented on src/wp-admin/about.php:
```diff
@@ -53,429 +53,155 @@
-				</p>
-				<p><?php _e( 'A new modal design offers a background blur effect, making it easier to focus on the task at hand.' ); ?></p>
+				<h3><?php _e( 'Experience the Site Editor, now out of beta' ); ?></h3>
+				<p><?php _e( 'Stable and ready for you to dive in and explore: 6.2 is your personal invitation to discover what the next generation of WordPress—and Block themes—can do.' ); ?></p>
```
Maybe some missing spaces here?
`WordPress — and Block themes — can do`

audrasjb replied:
well, no it looks good in en_US
https://prowritingaid.com/grammar/1000066/Should-I-use-spaces-around-an-em-dash

ryelle replied:
Looks like without spaces is part of the WP brand style: ["Use an em dash (—) without spaces."](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/#punctuation)

richtabor commented:
LGTM. We can iterate from here.

ryelle commented:
Committed in [r55449](https://core.trac.wordpress.org/changeset/55449).
